src/index.ts

```diff
@@ -22,12 +22,22 @@ dotenv.config();
 const logger = Container.get(LoggerService);
 
 // Initialise the ripple-lib service
-Container.get(RippleLibService).init().then(() => {
-    logger.info("Connected to ripple");
-}).catch(() => {
-    logger.error("Connecting to ripple failed");
-    process.exit(0);
-});
+let retries = 0;
+const xrpConnection = setInterval(function() {
+    if (parseInt(process.env.MAX_RECONNECT) > retries) {
+        Container.get(RippleLibService).init().then(() => {
+            logger.info("Connected to ripple!");
+            clearInterval(xrpConnection);
+        }).catch((e) => {
+            retries += 1;
+            logger.error(e);
+            logger.error(`Connecting to ripple failed! (attempt: ${retries})`);
+        });
+    } else {
+        logger.error(`Connecting to ripple failed ${process.env.MAX_RECONNECT} times! (stopping app)`);
```

Comment:
I get that process.env.MAX_RECONNECT is undefined when it has tried to reconnect 10 times

Reply:
did you add the MAX_RECONNECT variable to your local .env file ?

Reply:
Ah might be it